### PR TITLE
Json Serializer deprecation

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to https://github.com/Particular/PlatformDevelopment/issues/1374

### CODE REVIEW

File: nservicebus/upgrades/6to7/index.md [hunk 1]
```diff
@@ -166,6 +166,13 @@ The [Message Property Encryption feature](/nservicebus/security/property-encrypt
 See the [NServiceBus.Encryption.MessageProperty upgrade guide](/nservicebus/upgrades/externalize-encryption.md) for more details.
 
 
+## Json serialization
+
+The [Json serializer](/nservicebus/serialization/json/index.md) has been moved from the NServiceBus package. Use the external Json serializer available as a separate NuGet package, [NServiceBus.Newtonsoft.Json ](https://www.nuget.org/packages/NServiceBus.Newtonsoft.Json /).
```
Comment (SimonCropp) [inline nservicebus/upgrades/6to7/index.md]: > The [Json serializer](/nservicebus/serialization/json/index.md) has been moved from the NServiceBus package

technically this did not happen in v7. in v7 the change is "we removed the merged json serializer" but the alternative externalone has existed for years

**Comment by SimonCropp** [comment]
perhaps deep link here https://docs.particular.net/nservicebus/serialization/newtonsoft#compatibility-with-the-core-json-serializer

**Comment by SimonCropp** [comment]
i would not link to the external nuget. link to the external newtonsoft doco since we want them to read the doco prior to "just move nugets"

File: nservicebus/upgrades/6to7/index.md (hunk 1 above)
Comment (danielmarbach) [inline nservicebus/upgrades/6to7/index.md]: Yeah but not sure that level of detail is necessary.

**Comment by danielmarbach** [comment]
@SimonCropp I think this is good to go. I saw that https://docs.particular.net/nservicebus/messaging/third-party-integration mentions the json serializer as well but I came to the conclusion this page does not require any special warnings or additional information that mentions the json serializer being moved out of the core

**Comment by SimonCropp** [comment]
Seems content validation is failing?

**Comment by danielmarbach** [comment]
Stupid me. Pushed an update

**Comment by danielmarbach** [comment]
@timbussmann @bording quick review?

File: nservicebus/serialization/json.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 reviewed: 2016-08-23
 component: Json
 related:
  - samples/serializers/json
 ---
 
-Using [Json](https://en.wikipedia.org/wiki/Json) via an [ILMerged](https://github.com/Microsoft/ILMerge) copy of [Json.NET](http://www.newtonsoft.com/json).
+Using [JSON](https://en.wikipedia.org/wiki/Json) via an [ILMerged](https://github.com/Microsoft/ILMerge) copy of [Json.NET](http://www.newtonsoft.com/json).
+
+WARNING: In Versions 7 and above of NServiceBus, the JsonSerializer has been removed. Use the external [Json.NET Serializer](/nservicebus/serialization/newtonsoft.md) instead. See also [Compatibility between the core and external Json.NET serializer](/nservicebus/serialization/newtonsoft.md#compatibility-with-the-core-json-serializer).
```
Comment (bording) [inline nservicebus/serialization/json.md]: Would it make more sense to have this before the line above?

**Comment by bording** [comment]
@Particular/docs-maintainers What else do we need to do here to make sure that NSB 7.x won't show up in the dropdown for this page?

**Comment by SimonCropp** [comment]
@bording https://github.com/Particular/docs.particular.net/pull/2955/commits/ee1ebdb3d6044e60d2dcc6d0bb970e1004228faa

**Comment by danielmarbach** [comment]
THx for the review and help here!

**Pull request merged**